pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -804,3 +785,24 @@ def get_coinc_snr(trigs_or_injs, ifos):
     coinc_snr = numpy.sqrt(snr_sum_square)
 
     return coinc_snr
+
+
+def template_hash_to_id(trigger_file, bank_path):
+    """
+    This function converts the template hashes from a trigger file
+    into 'template_id's that represent indices of the
+    templates within the bank.
+    Parameters
+    ----------
+    trigger_file: h5py File object for trigger file
+    bank_file: filepath for template bank
+    """
+    bank = h5py.File(bank_path, "r")
+    hashes = bank['template_hash'][:]
```

Comment:
```suggestion
    with h5py.File(bank_path, "r") as bank:
        hashes = bank['template_hash'][:]
```
so you close the bank file as soon as you are done with it.